cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -913,6 +914,21 @@ function Configure-CniNetworking {
   Log-Output "CNI config:`n$(Get-Content -Raw ${l2bridge_conf})"
 }
 
+# Obtain the host dns conf and save it to a file so that kubelet/CNI
+# can use it to configure dns suffix search list for pods.
+# This function must be called after Configure-HostNetworkingService.
+function Configure-HostDnsConf {
+  $net_adapter = Get_MgmtNetAdapter
+  $server_ip = (Get-DnsClientServerAddress `
+          -InterfaceAlias ${net_adapter}.Name).ServerAddresses
```

Comment:
`ServerAddresses` suggests that this could be an array of addresses... is that possible / do we need to account for that?

Reply:
The value of DNS server is ignored right now because the pod will always only use cluster DNS service. But for consistency, I updated the patch so that it is in the same format as linux resolve conf.

Reply:
Great, thanks. Consider adding your note as a comment here: "The value of DNS server is ignored right now because the pod will always only use cluster DNS service, but ..."

Reply:
Good suggestion. Updated to add the comment.